README.adoc

```diff
@@ -180,34 +280,43 @@ server.xml
 include::finish/system/src/main/liberty/config/server.xml[]
 ----
 
-The `system` service is hardcoded to use a single forward slash as the context
-root. The context root is set in the [hotspot=webApplication
-file=0]`webApplication` element where the `contextRoot` attribute is specified
-as `"/"`. You'll make the value of the `contextRoot` attribute configurable by
+The `system` service is hardcoded to use a single forward slash as the context root.
+The context root is set in the [hotspot=webApplication file=0]`webApplication`
+element where the `contextRoot` attribute is specified as `"/"`.
+You'll make the value of the `contextRoot` attribute configurable by
 implementing it as a variable.
 
-[role="code_command hotspot", subs="quotes"]
+[role="code_command hotspot file=0", subs="quotes"]
 ----
 #Replace the `server.xml` file.#
 `system/src/main/liberty/config/server.xml`
 ----
 
-The `contextRoot` attribute in the [hotspot=webApplication
-file=0]`webApplication` element now gets its value from the
-[hotspot=context.root file=0]`context.root` variable. To find a value for the
-`context.root` variable, Open Liberty will look for the following environment
-variables, in order:
+The `contextRoot` attribute in the [hotspot=webApplication file=0]`webApplication`
+element now gets its value from the [hotspot=context.root file=0]`context.root` variable.
+To find a value for the `context.root` variable,
+Open Liberty will look for the following environment variables, in order:
```

Comment:
Open Liberty will look for the following environment variables
--->
Open Liberty looks for the following environment variables